src/status_im/translations/sv.cljs

```diff
@@ -55,180 +76,300 @@
    :datetime-yesterday                    "igår"
    :datetime-today                        "idag"
 
-   ;profile
+   ;;profile
    :profile                               "Profil"
+   :edit-profile                          "Redigera profil"
    :message                               "Meddelande"
    :not-specified                         "Inte angivet"
    :public-key                            "Offentlig nyckel"
    :phone-number                          "Telefonnummer"
+   :update-status                         "Uppdatera din status..."
+   :add-a-status                          "Lägg till status..."
+   :status-prompt                         "Lägg till din status. Använd #hashtags för att göra det lättare för andra att hitta dig och tala om det som intresserar dig"
    :add-to-contacts                       "Lägg till i kontakter"
+   :in-contacts                           "I kontakterna"
+   :remove-from-contacts                  "Ta bort från kontakter"
+   :start-conversation                    "Börja en konversation"
+   :send-transaction                      "Sänd en transaktion"
+   :testnet-text                          "Du är på {{testnet}} Testnet. Sänd inte äkta ETH eller SNT till din address"
```

Comment:
Istället för äkta, riktiga kanske? Äkta känns mer som "authentic". Samma nedan isåfall.